app/src/main/java/com/guzelgimadieva/tasky/authorization/ui/LoginEvent.kt

```diff
@@ -0,0 +1,18 @@
+package com.guzelgimadieva.tasky.authorization.ui
+
+sealed interface LoginEvent {
+
+    data class EmailChanged(
+        val email: String
+    ) : LoginEvent
+
+    data class PasswordChanged(
+       val password: String
+    ) : LoginEvent
+
+    data class PasswordVisibilityChanged(
+        val isPasswordVisible: Boolean
+    ) : LoginEvent
```

Comment:
Since the visibility will always change from true to false or false to true, it's not necessary to provide the boolean value here, since the VM can just negate the current value